src/WindowsInstaller/install.cmd

```diff
@@ -3,7 +3,7 @@ setlocal
 
 echo Executing install.cmd at %date% %time%
 
-set DATADOG_APPCMD_CMDLINE=%systemroot%\system32\inetsrv\appcmd.exe set config /section:system.webServer/modules /+[name='DatadogTracingModule',type='Datadog.Trace.AspNet.TracingHttpModule,Datadog.Trace.AspNet,Version=1.0.0.0,Culture=neutral,PublicKeyToken=def86d061d0d2eeb',preCondition='managedHandler,runtimeVersionv4.0']
+set DATADOG_APPCMD_CMDLINE=%systemroot%\system32\inetsrv\appcmd.exe set config /section:system.webServer/modules /+[name='OpenTelemetryInstrumentationModule',type='Datadog.Trace.AspNet.TracingHttpModule,OpenTelemetry.Instrumentation.AspNet,Version=1.0.0.0,Culture=neutral,PublicKeyToken=def86d061d0d2eeb',preCondition='managedHandler,runtimeVersionv4.0']
```

Comment:
@dszmigielski this should be the cause of the IIS failures: it should be "OpenTelemetry.AutoInstrumentation.AspNet"